cds_ils/migrator/eitems/api.py

```diff
@@ -222,19 +238,27 @@ def migrate_ezproxy_links():
         click.echo("Processing document {}...".format(document["pid"]))
 
         for url in document["_migration"]["eitems_external"]:
-            eitem = create_eitem(document["pid"], open_access=False)
-            url["login_required"] = True
-            eitem["urls"] = [url]
-            eitem.commit()
-            EItemIndexer().index(eitem)
+            try:
+                eitem = create_eitem(document["pid"], open_access=False)
+                url["login_required"] = True
+                eitem["urls"] = [url]
+                eitem.commit()
+                EItemIndexer().index(eitem)
```

Comment:
I would avoid all the hardcoding of the classes around, just in case the import is not correct... but that's ok at this point.